### ISSUE TITLE
I guess the project is dead

### ISSUE THREAD

**Issue by panosru** [firecore/Seas0nPass#18]
From my understanding, the project seems to be dead, no?

MacOS High Sierra 10.13.6 (I tried with Mojave on my second machine and also Sierra and El Capitan on virtual machine).

![image](https://user-images.githubusercontent.com/400362/61283089-391b5800-a7c5-11e9-878c-5e577fee5d3e.png)

**Comment by lechium** [comment]
yes this project is dead. getting it to work on the last version that it did (i think either el cap or mojave) was a huge ordeal. apple breaks their usb stack every year. maintaining jailbreaks on newer devices is enough work as is, devices this old arent likely to get my attention again. sorry.

**Comment by panosru** [comment]
That is understandable, what isn't understandable is why firecore still [allows users to buy a discontinued product](https://firecore.com/atvflash-black).

![image](https://user-images.githubusercontent.com/400362/61445194-9514e680-a955-11e9-91e4-6f6c073e9df1.png)

Not only that, but on the requirement section it does not mention that the product they sell is not compatible with High Sierra (10.13) and later

![image](https://user-images.githubusercontent.com/400362/61445443-153b4c00-a956-11e9-9a82-bc15e0b94a65.png)

I have bought that product exactly one year ago and 1st I never got the support I needed to make it work, 2nd it wasn't supported in the first place even when I bought it, as a consumer what should I make of that?

**Comment by lechium** [comment]
im really sorry, i only contracted for firecore so i dont officially speak for the company, i wish i had better answers for you.
